src/arrow_reader.rs

```diff
@@ -887,7 +887,13 @@ pub struct StreamMap {
 }
 
 impl StreamMap {
-    pub fn get(&self, column: &Column, kind: Kind) -> Option<Decompressor> {
+    pub fn get(&self, column: &Column, kind: Kind) -> Result<Decompressor> {
+        self.get_opt(column, kind).context(InvalidColumnSnafu {
+            name: column.name(),
+        })
+    }
```

Comment:
Instead of having to handle the case where column stream doesn't exist everywhere we call, just centralize the err here